examples/basic-with-tests-cucumber/build.sbt

```diff
@@ -0,0 +1,49 @@
+
+
+name := "basic-cucumber"
```

Comment:
It would be great if you could also update the `README.md` to include an [example #8](https://github.com/Tapad/sbt-docker-compose#examples) covering cucumber.